package.json

```
@@ -55,7 +55,7 @@
     "prettier": "^2.5.0"
   },
   "engines": {
-    "node": "16",
+    "node": ">=16.x.x <=18.x.x",
     "npm": "8"
```

Per my comment in the issue:

```suggestion
    "npm": ">=8"
```

---

https://github.com/strapi-community/strapi-provider-upload-google-cloud-storage/issues/155#issuecomment-1412832543